src/test/resources/unrc/dose/Bono.feature

```diff
@@ -0,0 +1,91 @@
+
+Feature: test based challenge equals
+     Scenario Outline: User solves an unsolved test based challenge.
+Given that user <user> is already logged on 
+And the user wants to solve the test based challenge
+And the user has already solve this challenge before with a score of 12 points
```

Comment:
This condition seems to contradict the scenario outline, specifically that the test based challenge is unsolved, but this condition states that the challenge was solved with a score of 12 points.